src/main/archetype/ui.content/src/main/content/jcr_root/conf/__confFolderName__/settings/wcm/templates/category-page/initial/.content.xml

```diff
@@ -20,7 +20,8 @@
                     jcr:lastModified="{Date}2019-03-26T14:33:35.239+02:00"
                     jcr:lastModifiedBy="admin"
                     jcr:primaryType="nt:unstructured"
-                    sling:resourceType="${appsFolderName}/components/commerce/productlist"/>
+                    sling:resourceType="${appsFolderName}/components/commerce/productlist"
+                    cq:styleIds="[1570700958900]" />
```

Comment:
If you apply the style to the initial page template it will only be used by new pages created. Should we apply it also to the existing category page in language master & us sites?

Reply:
Good point, I'll update that 👍